packages/components/src/components/Table/Table.tsx

```diff
@@ -36,15 +36,25 @@ const Table = React.forwardRef<HTMLTableElement, TableProps>(
 
     return (
       <TableContext.Provider value={tableContext}>
-        <StyledTable
-          borderCollapse="separate"
-          ref={ref}
-          tableLayout={tableLayout}
-          w="100%"
-          {...props}
+        <Box.div
```

Comment:
The reason to put this wrapper is the border and background, right? What if we are capable to add these styles not adding a div to it? My concern here is to wrap the table around an element to solve something that could be made by pure CSS

Reply:
Could we put it in the styled table?

Reply:
Ana showed me the troubles with sticky headers not having the wrapper and life sometimes can be hard

Reply:
It doesn't work as expected if we add to the StyledTable just because of the Sticky variant. :( 

In the video, you can see that when I scroll down, the table goes up and therefore the border top and bottom disappear because they are part of the Table. That's why I added the wrapper. 
https://github.com/Localitos/pluto/assets/5320963/ec8fc0c3-93bc-44d5-95aa-766904461362